sdk/src/connections/network/buffer.proto

```diff
@@ -36,20 +36,30 @@ message SensorsInfo
 }
 
 // TO DO: move this to a common file to be shared with USB
-message DepthSensorFrameContent
+message DriverConfiguration
 {
-  string type = 10;
-  uint32 width = 20;
-  uint32 height = 30;
+  string depthBits = 10;
+  string abBits = 20;
+  string confBits = 30;
+  string pixelFormat = 40;
+  int driverWidth = 50;
+  int driverHeigth = 60;
+  int pixelFromatIndex = 70;
 }
 
 // TO DO: move this to a common file to be shared with USB
 message DepthSensorFrameType
 {
-  string type = 10;
-  repeated DepthSensorFrameContent depthSensorFrameContent = 20;
-  uint32 width = 30;
-  uint32 height = 40;
+  string mode = 10;
+  repeated string frameContent = 20;
+  uint8_t modeNumber = 30;
+  int pixelFromatIndex = 40;
+  int frameWidthInBytes = 50;
+  int frameHeightInBytes = 60;
+  int baseResolutionWidth = 70;
+  int baseResolutionHeight = 80;
+  int metadataSize = 90;
+  repeated DriverConfiguration driverConfiguration = 100;
```

Comment:
Is this meant to be used in the future? If yes, should 'repeated' be used?

Reply:
Repeated is used because a vector of DriverConfiguration will be used. 
https://github.com/analogdevicesinc/ToF/blob/add_tables_and_selection_class/sdk/src/connections/target/sensor-tables/driver_configuration_table.h#L112